### DISCUSSION TITLE
Languages plugin skipping lots of commits

### DISCUSSION THREAD

**Discussion by Nixinova** [lowlighter/metrics#1207] [Q&A]
Sometime between Aug 28 and Sep 2 metrics has now ignored a lot of commits when fetching language stats.

Action runs (note the time diff):
![image](https://user-images.githubusercontent.com/42429413/188339595-6be901eb-71f5-4cfb-9241-083da83a8bc3.png)

3 days ago output:
![image](https://user-images.githubusercontent.com/42429413/188339607-35619925-8c4c-49b2-abfe-9fa840602fe8.png)

7 days ago output:
![image](https://user-images.githubusercontent.com/42429413/188339613-000d865e-b9fa-4b74-821e-8c6d97f9ae6d.png)

**Comment by lowlighter** [comment]
Hi 👋 !

Hum, the [`languages/analyzers.mjs`](https://github.com/lowlighter/metrics/blob/master/source/plugins/languages/analyzers.mjs) hasn't been touched yet for a long time, so it could be an external factor

I don't remember if there was some `git` update recently, but possible some output changed ?

Anyway, I guess when #1118 eventually lands hopefully this mode will be more reliable 😅
